### ISSUE TITLE
CPU Usage

### ISSUE THREAD

**Issue by webfrank** [kubenetworks/kubevpn#197]
Hi, over time the kubevpn traffic manager pod increases the CPU usage, I need to restart it in order to lower CPU usage. This happens regardless of using or not the VPN.

**Comment by wencaiwulue** [comment]
> Hi, over time the kubevpn traffic manager pod increases the CPU usage, I need to restart it in order to lower CPU usage. This happens regardless of using or not the VPN.

hello，sorry to hear that. Which version are you using？. Maybe progarm have bug...

**Comment by wencaiwulue** [comment]
> Hi, over time the kubevpn traffic manager pod increases the CPU usage, I need to restart it in order to lower CPU usage. This happens regardless of using or not the VPN.

Yes, i found the reason, it's because function `time.Tick`, in loop to call `time.Tick` is bad way. 
<img width="992" alt="image" src="https://github.com/kubenetworks/kubevpn/assets/33818115/92c83b5f-fd34-447b-9724-fc5cbadd0c0d">

<img width="1272" alt="image" src="https://github.com/kubenetworks/kubevpn/assets/33818115/58f132c2-4594-493e-9ced-d94a25574b41">

I will change it to `time.NewTicker`, just like:
```
func() {
  for {
    select {
      case <-time.Tick(10*time.Seconds):
    }
  }
}
```
to
```
func() {
  ticker := time.NewTicker(10*time.Seconds)
  for {
    select {
      case <- ticker.C:
    }
  }
}
```

**Comment by wencaiwulue** [comment]
@webfrank already released new version v2.2.4, fixed this bug, you can have a try ~ (using helm to install server: [ref](https://www.kubevpn.cn/docs/quickstart/))

**Comment by webfrank** [comment]
Hi, I left running it for some days now and it seems good so far. 

Thank you for the rapid fix.
